pkg/uploader/kopia/snapshot.go

```diff
@@ -29,8 +29,8 @@ import (
 	"github.com/sirupsen/logrus"
 
 	"github.com/vmware-tanzu/velero/pkg/repository/udmrepo"
+	"github.com/vmware-tanzu/velero/pkg/repository/udmrepo/kopialib/logging"
```

Comment:
As the design of Unified Repository, this kopia uploader should be able to write data to any repository that implements Unified Repo Interface. So the kopia uploader should not know the existence of `kopialib` or depend on it, as `kopialib` is the implementation of Unified Repo Interface for kopia only.

Therefore, it is not suitable to move kopia_log into `kopialib`. One suggestion is to add a `kopia` folder under `pkg` like what we do for restic. Or the final path is `pkg/kopia/kopia_log.go`

Reply:
Addressed.